transfering-a-cookbook.md

```diff
@@ -0,0 +1,38 @@
+# Transferring a Cookbook to the Chef Brigade
+
+Got a cookbook you'd like help with? We'd love to help!
+
+## Before you begin
+
+We need to work with the github repo owner and supermarket cookbook owner. If you aren't this person, let us know and we'll try to contact them. If necessary, fine beverages and chocolate chip cookies may be provided to encourage them to work with us.
+
+You'll need:
+
+- Code Repository URL
+- Supermarket cookbook name
+
+## Transferring the code
+
+We're assuming the cookbook code is already in its own repo in GitHub. If it's in a shared repo, or hosted elsewhere, it'll require a slightly different approach.
```

Comment:
It depends?

There's the script to extract a cookbook's history from a monolithic repo. Then upload to new repo under brigade. 

For non-github repos, extract to a local repo, add github remote, push. 

I mostly left this vague because there are cases we won't have considered.